internal/util/function/rerank/function_score.go

```diff
@@ -159,11 +159,11 @@ func createFunction(collSchema *schemapb.CollectionSchema, funcSchema *schemapb.
 	return rerankFunc, nil
 }
 
-func NewFunctionScore(collSchema *schemapb.CollectionSchema, funcScoreSchema *schemapb.FunctionScore) (*FunctionScore, error) {
+func NewFunctionScore(collSchema *schemapb.CollectionSchema, funcScoreSchema *schemapb.FunctionScore, extralInfo map[string]string) (*FunctionScore, error) {
```

Comment:
nit: using str map to pass structured info is a little bit tricky. Turning into a struct will be better. 
Just a small point, not that critical